.gitignore

```diff
@@ -2,3 +2,4 @@
 node_modules/
 .idea
 .nyc_output
+.tap/
```

Comment:
Why?

Reply:
Shouldn't we keep the tests artifacts from `tap` out of repository?

Reply:
@Fdawgs 

does tap generates artifacts?

Reply:
Here is what I am getting after running `npm test`

```bash
> git status
On branch main
Your branch is behind 'origin/main' by 2 commits, and can be fast-forwarded.
  (use "git pull" to update your local branch)

Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git restore <file>..." to discard changes in working directory)
	modified:   README.md

Untracked files:
  (use "git add <file>..." to include in what will be committed)
	.tap/

no changes added to commit (use "git add" and/or "git commit -a")
```

![image](https://github.com/fastify/github-action-merge-dependabot/assets/3785575/4aee5600-8044-4e8e-b4fc-5e1866a06f36)

Reply:
Thats strange, because we have a general .gitignore file and if tap generates artifacts we should also cover them in that file. Thats why I am pinging @Fdawgs

Reply:
@Uzlopak The general .gitignore doesn't have this as I don't believe any other Fastify repo generates tap artefacts.

Reply:
I am happy to remove the `.tap/` from the ignore and just not adding the artifacts to the PR. This PR is really just about the doc/readme. And I don't want to introduce additional confusion here. 
I guess the topic of what to put in gitignore can be solved elsewhere. WDYT @Uzlopak @Fdawgs ?

Reply:
Probably best to just add your doc changes, thanks @groozin.